qt/aqt/gui_hooks.py

```diff
@@ -11,3 +11,5 @@
 
 # You can find the definitions in ../tools/genhooks_gui.py
 from aqt.hooks_gen import *
+
+
```

Comment:
These could be removed; otherwise looks good thanks. Rebasing onto master again should fix the Windows tests not working

Reply:
Done.

Reply:
I'm still getting a failure on Windows it seems.